### ISSUE TITLE
nsenter regression using --all on older kernels

### ISSUE THREAD

**Issue by ringerc** [util-linux/util-linux#2059]
The `--all` command line option of `nsenter` versions is not compatible with older Linux kernels, and will fail with e.g.

```
root# nsenter -t 1 --all
nsenter: cannot open /proc/1/ns/time: No such file or directory
```

This appears to have been caused by commit 7f1f0584c24a77909a7c96e62e30f63f4c1b10ad 

```
commit 7f1f0584c24a77909a7c96e62e30f63f4c1b10ad
Author: Adrian Reber <areber@redhat.com>
Date:   Mon Mar 9 13:02:45 2020 +0000

    nsenter: add support for the time namespace
    
    Just as with unshare and lsns this adds time namespace support to
    nsenter.
    
    In contrast to unshare nsenter does not have the options '--boottime'
    and '--monotonic' as that offset can only be set as long as there have
    no processes being started in the corresponding time namespace.
    
    Signed-off-by: Adrian Reber <areber@redhat.com>
```

**Comment by ringerc** [comment]
This issue has already been fixed by

```
commit 2f628e8f9ff7b72c4041a79bf3ff28f214cafb6f
Author: Yonatan Goldschmidt <yon.goldschmidt@gmail.com>
Date:   Fri Nov 12 15:27:51 2021 +0200

    nsenter: Do not try to enter nonexisting namespaces when --all is used
    
    This PR changes is_same_namespace() to is_usable_namespace(). If a namespace file
    is missing for the running PID, then it is considered not usable and we do not try
    to enter it.
    
    Signed-off-by: Yonatan Goldschmidt <yon.goldschmidt@gmail.com>
```

The `nsenter` in Ubuntu 22.04 seems to be between the two versions, so it'll break.

**Comment by ringerc** [comment]
Fix commit is 2f628e8f9ff7b72c4041a79bf3ff28f214cafb6f

**Comment by ringerc** [comment]
At time of writing there was no tagged release including the above commit; it'll land in the release after the currently-latest https://github.com/util-linux/util-linux/releases/tag/v2.38.1
